caravel/assets/javascripts/explorev2/components/FieldSet.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PropTypes } from 'react';
```

Comment:
i called it `FieldSet` because it renders the field and the label so it's a set in my mind....